### PR TITLE
{data}[intel/2015a] h5py 2.5.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9623f5f9150be75472e9 for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015a-Python-2.7.9-gpfs.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name = 'h5py'
+version = '2.5.0'
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': False}
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015a-Python-2.7.9-gpfs.eb]: you should use the serial version of HDF5?

Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015a-Python-2.7.9-gpfs.eb]: or set this to True :)

I'll do that.

Thanks for catching this.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e01a737ab0158e2e8489 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**